src/group/Group.cc

```diff
@@ -356,84 +356,85 @@ void Group::add_admin_rules(int user_id)
 
     // #<uid> USER/@<gid> USE+MANAGE+ADMIN+CREATE *
     if ( aclm->add_rule(
-            AclRule::INDIVIDUAL_ID |
-            user_id,
+                 AclRule::INDIVIDUAL_ID |
+                 user_id,
 
-            PoolObjectSQL::USER |
-            AclRule::GROUP_ID |
-            oid,
+                 PoolObjectSQL::USER |
+                 AclRule::GROUP_ID |
+                 oid,
 
-            AuthRequest::USE |
-            AuthRequest::MANAGE |
-            AuthRequest::ADMIN |
-            AuthRequest::CREATE,
+                 AuthRequest::USE |
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

We observed that you are using unintended operations or assignments that could lead to logical errors, vulnerabilities, or undefined behavior. To prevent these issues, ensure that you follow best practices, such as using explicit comparisons and clear operator usage, and using parentheses to clarify intent. Learn more - https://cvefeed.io/cwe/detail/cwe-480-use-of-incorrect-operator